forc-util/src/lib.rs

```diff
@@ -409,6 +409,8 @@ fn construct_window<'a>(
     &input[calculated_start_ix..calculated_end_ix]
 }
 
+/// A subscriber built from default tracing_subscriber::fmt::SubscriberBuilder such that it would match directly using println! throughout the repo
+/// RUST_LOG environment variable can be used to set different minimum level for the subscriber, default is INFO
```

Comment:
I meant for these docs to go on the `init_tracing_subscriber` function :smile: This `const` is not `pub`, so these docs would never be exposed from the crate docs.

Reply:
Ohh my bad, sorry